toml/src/main/kotlin/org/rust/toml/completion/crates/local/CratesLocalIndexState.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.completion.crates.local
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.util.xmlb.XmlSerializerUtil
+
+@State(name = "CratesLocalIndexState", storages = [Storage("rust.crateslocalindex.xml")])
+class CratesLocalIndexState : PersistentStateComponent<CratesLocalIndexState> {
```

Comment:
Probably it's better to make `CratesLocalIndexService : PersistentStateComponent` and put `CratesLocalIndexState` as a data class inside the service. Discussed in person.